mainwindow.ui

```diff
@@ -6,8 +6,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>1046</width>
-    <height>935</height>
+    <width>1134</width>
+    <height>1015</height>
    </rect>
```

Comment:
This was unintentional. You might want to redo the UI yourself, I have a hard time getting the results I want with the Designer-Tool :(

Reply:
okay, this is not a big problem in my view